virt/virtual_machines/vm_networking/virt-hot-plugging-network-interfaces.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: ASSEMBLY                                     
+[id="virt-hot-plugging-network-interfaces"]                            
+= Hot plugging secondary network interfaces                                            
+include::_attributes/common-attributes.adoc[]                  
+:context: virt-hot-plugging-network-interfaces                        
+                                                                
+toc::[]     
+
+You can add or remove secondary network interfaces without stopping your virtual machine (VM). {VirtProductName} supports hot plugging and hot unplugging for Linux bridge interfaces that use the VirtIO device driver.
+
+:FeatureName: Hot plugging and hot unplugging bridge network interfaces
+include::snippets/technology-preview.adoc[]
+
+[id="virtio-limitations_{context}"]
```

Comment:
I might be mistaken but I think a peer reviewer recently told me that anchor IDs in assemblies should not contain the context attribute.